text_extensions_for_pandas/array/span.py

```diff
@@ -466,6 +466,17 @@ def __hash__(self):
                                self._ends.tobytes()))
         return self._hash
 
+    def __contains__(self, item) -> bool:
+        """
+        Return true if scalar item exists in this SpanArray.
+        :param item: scalar Span value.
+        :return: true if item exists in this SpanArray.
+        """
+        if isinstance(item, Span) and \
+                item.begin == Span.NULL_OFFSET_VALUE:
+            return Span.NULL_OFFSET_VALUE in self._begins
```

Comment:
This is needed because otherwise the default is to check all values for equality, and that will fail if there is a NULL value with `Span.target_text=None`

Reply:
Looks like the current Pandas policy is "¯\_(ツ)_/¯"; see the warning on https://pandas.pydata.org/pandas-docs/stable/user_guide/missing_data.html

Reply:
Yeah, good question. The test failure related to this was checking `assert na_value in data_missing` which expects that `na_value == na_value` to be True. Seems like it would be different for `np.nan` though..